doc/design/show_train_stmt.md

```diff
@@ -0,0 +1,52 @@
+# Show Train Statement in SQLFlow
+
+## Background
+
+Like using other SQL engines, people may want to access the table metadata (say, the  table creating statement) using a SQL query. Usually they will type a statement like:
+```
+SHOW CREATE TABLE `test`;
+```
+which will produce:
+```
+mysql> SHOW CREATE TABLE `test`;
++-------+------------------------------------------------------------------+
+| Table | Create Table                                                                                                                                                                                                                                                          |
++-------+------------------------------------------------------------------+
+| test  | CREATE TABLE `test` (
+  `sepal_length` float DEFAULT NULL,
+  `sepal_width` float DEFAULT NULL,
+  `petal_length` float DEFAULT NULL,
+  `petal_width` float DEFAULT NULL,
+  `class` int DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
++-------+-----------------------------------------------------------------+
+```
+
+It worth for SQLFlow to do the same thing. On SQLFlow platform, user may create a lot of models with a minor set of parameters different with each other. After a while, the  user may hardly tell what the differences are if he hadn't record the information else where (like in model name).  We would like to provide a formal way to solve this problem. Here we introduce the SHOW TRAIN feature.
+
+## Grammar
+```
+SHOW TRAIN table_where_model_stored;
```

Comment:
The train clause use `TO TRAIN` so maybe we should use `SHOW TO TRAIN` here to be consistent.

Reply:
As I see, SHOW TRAIN is more like a standard SQL stmt, which may lead to a more consistent user experience for analysts. So I still prefer not use a 'TO' here.

Reply:
Just for discussion, `SHOW CREATE TABLE [table_name]`  is showing statement `CREATE TABLE [table_name] ...`, yet `SHOW TRAIN [trained_model_name]` is showing statement `SELECT ... TO TRAIN ... INTO [trained_model_name]`. Maybe not straight forward, I'd prefer `DESCRIBE MODEL [trained_model_name]` to show all model metadatas like training statement as well as column information generated by feature derivation.

Reply:
We may implement this feature later, as I see, current serialized model has no column info.